_posts/2020-12-10-webpack-vs-sprockets.markdown

```diff
@@ -0,0 +1,313 @@
+---
+layout: post
+title: "Webpack VS Sprockets"
+date: 2020-11-13 14:00:00
+categories: ["learning", "webpack"]
+author: arieljuod
+---
+
+Since the release of Rails 6, **Webpack** is the default JavaScript bundler for new Rails app. We all struggle at first coming from a **Sprockets** background, and more often than not, we, as developers, tried to avoid making JavaScript changes so we don't have to deal with it, or just copy/pasted until it worked.
```

Comment:
Who is the "we" here? Do you mean, we as ruby developers?, we are rails developers? The tenses in the sentence are mixed a little. Maybe:
```suggestion
Since the release of Rails 6, **Webpack** is the default JavaScript bundler for new Rails apps. We all struggled at first coming from a **Sprockets** background, and more often than not, we, as developers, tried to avoid making JavaScript changes so we wouldn't have to deal with it, or just copy/pasted until it worked.
```
Also, I would consider removing the last part of the sentence, I don't think it's necessary. But that's up to you. "or just copy/pasted until it worked."

Reply:
I mean we as rails developers. Maybe I should just use `we` both times with no `as developers`? I didn't want it to sound like `we at ombulabs`.

Reply:
I think the best would be "we, as Rails developers"